deploy/playbooks/03_app.yml

```
@@ -41,10 +44,10 @@
       when: not env_file.stat.exists
 
     - name: Start docker compose to see if everything is running
-      shell: "docker compose up -d"
+      ansible.builtin.shell: "docker compose up -d"
```

```suggestion
      ansible.builtin.command:
        chdir: {{ ansible_user_dir }}
        cmd: "docker compose up -d"
```
Please double check if this really works. chdir is only to make sure we are starting in the correct homedir might be absolutely unneccessary.